### ISSUE TITLE
Requesting Help - Save data using Pandas

### ISSUE THREAD

**Issue by SyedAlamdar** [nabeel-oz/qlik-py-tools#44]
So I know this library is not for this reason but I had a unique case where I am trying to save a variable value outside of Qlik. Let's say to_csv using panda. Can you please provide an example of where using panda we can export the data out and stage it in a particular folder. A good use case will be to triage events that are predicted. For a close loop to exist we can have other users evaluate the results and highlight false positives that will then again become part of the training model. 

In short, can we use python to extract the data out on to access to a destination of our choice? (and I am not talking about right-clicking and exporting the data). If you do intend to build a sample, would be nice to have a sample of how a variable value can be saved and if possible a table can be exported.

Regards,
Syed

**Comment by nabeel-oz** [comment]
For help on building your own SSE function you can refer to the Python examples at [Qlik-OSS](https://github.com/qlik-oss/server-side-extension#server-side-extension).

PyTools persists models to disk so you can also refer to that as an example. The [PersistentModel](https://github.com/nabeel-oz/qlik-py-tools/blob/master/core/_machine_learning.py) class defines methods to save and load a model from disk. 

As you can see this is a simple class. The communication with Qlik is abstracted away by other modules. I've used `joblib` in this case as the persistent model contains the trained sklearn estimator and related objects. If you were saving a pandas dataframe you could just save to csv.

**Comment by nabeel-oz** [comment]
Closing this issue as it has been inactive for ~2 months.
